content/SCALE/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Multiprotocol Shares"
+description: "Provides instructions on setting up SMB and NFSv4 mixed-mode shares."
+weight: 40
+tags:
+- scaleshares
+- scalesmb
+- scalenfs
+---
+
+{{< toc >}}
+
+## About Multiprotocol Shares
+
+A multiprotocol or mixed-mode NFS and SMB share supports both NFS and SMB protocols for sharing data.
+Multiprotocol shares allow clients running different operating systems to access the same data.
+This can be particularly useful in environments with a mix of Unix-like systems (which prefer NFS) and Windows systems (which prefer SMB).
+
+It is important to properly configure permissions and access controls to ensure security and data integrity when using mixed-mode sharing.
+To maximize security on the NFS side of the multiprotocol share, we recommend using NFSv4 and [Active Directory](#joining-active-directory) for Kerberos authentication.
+It is also important that NFS clients preserve extended attributes when copying files, or SMB metadata could be discarded in the copy.
+
+## Adding a Multiprotocol Share
+
+Adding a multiprotocol SMB and NFS share to your system involves several steps to add the share and get it working.
+
+1. [Configure Active Directory](#joining-active-directory) and join the TrueNAS SCALE system to your AD domain.
+   Configure a container, Kerberos admin, and user accounts in AD.
+
+2. [Set up a dataset](#creating-a-multiprotocol-share-dataset) for the new share with **Share Type** set to **Multiprotocol**.
+
+3. [Create the SMB share](#creating-the-smb-share) with **Purpose** set to **Multi-protocol (NFSv4/SMB) shares**.
+
+4. [Create the NFS share](#creating-the-nfs-share) with **Security** set to **KRB5**.
```

Comment:
There is a step involving NFS configuration where we have a button in the NFS form to add the NFS kerberos (KRB5) service principle name (SPN) to the AD machine account for the TrueNAS server in Active Directory. If KRB5 SPN is not present in local kerberos keytab and in AD machine account, then kerberized NFS clients will not be able to mount the NFS export.

Reply:
To address this I'm moving this step to the beginning and will include a tip that you will need to complete the form if AD is already joined.